utool/util_scripts/classfuncs.py

```diff
@@ -69,7 +69,7 @@ def get_codedir():
         'https://github.com/bluemellophone/detecttools.git',
         'https://github.com/Erotemic/hesaff.git',
         'https://github.com/bluemellophone/pyrf.git',
-        'https://github.com/Erotemic/ibeis.git',
+        'https://github.com/Erotemic/wbia.git',
```

Comment:
This and another line like it will likely break. I'd guess it's not terribly important to change as this likes like a casual script rather than a piece of the library to be used externally.